vitest.shims.d.ts

```diff
@@ -0,0 +1 @@
+/// <reference types="@vitest/browser/providers/playwright" />
```

Comment:
I'm not sure yet whether we really need this file. I also had to add some eslint overrides specifically for this file to prevent errors.

Will find out once I've tried out the test setup.